packages/superset-ui-legacy-plugin-chart-force-directed/src/ForceDirected.js

```diff
@@ -107,15 +116,22 @@ function ForceDirected(element, props) {
     .on('tick', tick)
     .start();
 
-  div.selectAll('*').remove();
-  const svg = div
-    .append('svg')
-    .attr('width', width)
-    .attr('height', height);
+  function resize() {
+    const { innerWidth, innerHeight } = window;
```

Comment:
Using `window` 's `innerWidth` and `innerHeight` makes the chart expand beyond its supposed size.